### ISSUE TITLE
Constellation plane disappears when I try to center on it

### ISSUE THREAD

**Issue by Chrondeath** [HellFirePvP/AstralSorcery#30]
Using the hand telescope; don't see a way to craft a standing telescope yet so I assume it's further in the mod.

Scanning around with the hand telescope, I can see the starfield, and I can find a location in the sky that appears to have a constellation based on seeing a separate plane of closer-looking stars that moves opposite to the telescope motion.  However, the plane of stars appears to be...stuck in the left-hand side of the telescope view.  If I attempt to move the view to put all the stars of the constellation in view, the constellation plane (the set of constellation stars) disappears just about when the rightmost star crosses the center of the view.  If I scan back the other direction, it reappears.  At the extreme I can see another star of the constellation peeking out from the side of the view, so I believe there are stars I don't have visible and thus can't draw the constellation without getting it centered.

**Comment by Chrondeath** [comment]
I created a world with the same seed in Creative mode, gave myself the hand telescope and a paper, and did not see the same problem; the constellation I was partially seeing showed up in the same location but wasn't blinking out when I tried to center it.  The problem still occurs in the survival world.  Not sure if it's an item or some mod's progression status or what....

**Comment by Chrondeath** [comment]
I tried unequipping and storing all my gear in the survival world except the hand telescope and the constellation paper.  The constellation behaved properly and didn't vanish.  Then I reequipped all my gear, and the constellation still behaved properly.  Then I did....nothing, clicked around in the UI a bit while some time passed, and when I looked for the constellation again it was disappearing again.  I'm very confused...

**Comment by HellFirePvP** [comment]
After reading this a couple of times, i _suppose_ i know where this could come from; however i thought i fixxed that as i came across such behavior once during development... Will see where what i can do and adress that.

**Comment by Chrondeath** [comment]
I played around with this a bit more last night and am still not sure what makes it occasionally work.  I thought it might be that loading into a creative world, looking at a constellation, and then loading into the broken world caused it to be fixed for some period of time; this worked on one attempt but not a second attempt.  (At one point there appeared to be a second bug, where the constellation itself showed up and was centerable but the starfield behind it was black and I couldn't draw lines to trace the constellation, but as it was right after loading in I was guessing it might be something not having been loaded yet.)

I was also suspicious that the Baubles UI or Botania's mana display might be throwing something off, which is why I was unequipping gear; I believe one test did have me seeing the bug, removing gear, and then not seeing the bug, without a restart, but I also remember trying that same sequence several times before and not having it fix anything.

After playing around with it I did take advantage of an instance of catching it fixed to trace the constellation, which should work to let me progress but I don't know if it means I won't be able to reproduce the issue any more (though I could restore a backup).
